src/layout/DwindleLayout.cpp

```diff
@@ -811,8 +837,40 @@ void CHyprDwindleLayout::alterSplitRatio(CWindow* pWindow, float ratio, bool exa
 }
 
 std::any CHyprDwindleLayout::layoutMessage(SLayoutMessageHeader header, std::string message) {
-    if (message == "togglesplit")
+    std::replace(message.begin(), message.end(), ' ', ',');
+    const auto ARGS = CVarList(message);
```

Comment:
this is cringe. Don't replace, just `CVarList(message, 0, ' ')`

Reply:
ah that's what you meant with the example